code/Bresenham Line Algorithm Programs/Line drawing algo.java

```diff
@@ -0,0 +1,145 @@
+
+package applet;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Graphics;
+import java.awt.event.MouseEvent;
+import java.awt.event.MouseListener;
+import java.awt.event.MouseMotionListener;
+
+public class bresenhamLine extends java.applet.Applet implements MouseListener, MouseMotionListener {
```

Comment:
Please call this class in UpperCamelCase. Also rename the file to that with a `.java` extension.